app/models/claim/advocate_claim.rb

```diff
@@ -180,16 +180,16 @@ def eligible_basic_fee_types
       # rather than conditionally return scheme 10 specifically
       # TBD once all the fee scheme work is integrated
       return Fee::BasicFeeType.all if from_api? || from_json_import?
-      return Fee::BasicFeeType.unscoped.agfs_scheme_10s.order(:position) if fee_scheme.eql?('fee_reform')
-      Fee::BasicFeeType.agfs_scheme_9s
+      return Fee::BasicFeeType.unscoped.agfs_scheme_10.order(:position) if fee_scheme.eql?('fee_reform')
+      Fee::BasicFeeType.agfs_scheme_9
     end
 
     def eligible_misc_fee_types
       # TODO: this should return a list based on the current given fee scheme
       # rather than conditionally return scheme 10 specifically
       # TBD once all the fee scheme work is integrated
-      return Fee::MiscFeeType.agfs_scheme_10s if fee_scheme == 'fee_reform'
-      Fee::MiscFeeType.agfs_scheme_9s
+      return Fee::MiscFeeType.agfs_scheme_10 if fee_scheme == 'fee_reform'
```

Comment:
just a comment as i do not understand how this effects the api but i rather like the plural version